### ISSUE TITLE
Loading mouse cursor is displayed after running Atom

### ISSUE THREAD

**Issue by aruseni** [atom/atom#12444]
### Description

When I open Atom, loading/waiting mouse cursor is displayed for about 30 seconds.

![loading](https://cloud.githubusercontent.com/assets/372098/17733561/ee9ff44c-647f-11e6-9d09-d354c077989a.png)
### Steps to Reproduce
1. Run Atom
2. Move the mouse cursor on either the header of any window, to desktop or to file manager

When I try running Atom in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode), I can’t reproduce this bug.

**Expected behavior:** Normal cursor

**Actual behavior:** For about 30 seconds you see loading/waiting cursor
### Versions

**atom --version**

```
Atom    : 1.9.8
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0
```

**apm --version**

```
apm  1.10.0
npm  2.13.3
node 0.10.40
python 2.7.12
git 2.7.4
```

**Comment by aruseni** [comment]
I am using Xubuntu 16.04.1 with Xfce 4.12.

**Comment by 50Wliu** [comment]
> When I try running Atom in safe mode, I can’t reproduce this bug.

That would suggest that one of your installed packages (and/or your custom init script) is causing the issue.  Try disabling packages until the issue goes away.

**Comment by aruseni** [comment]
Hi @50Wliu!

It happens on a fresh install of Atom, even if you disable all packages and do not do any config customizations.

**Comment by 50Wliu** [comment]
But safe mode still does not cause this error?  Is there any difference in how you're launching Atom with and without safe mode?

**Comment by aruseni** [comment]
Wow! Indeed, you are absolutely right, there is a difference in it.

Normally I used the “Whisker Menu” (standard menu in Xubuntu). And with this menu a URL is passed as first argument.

![screenshot_2016-08-19_19-05-14](https://cloud.githubusercontent.com/assets/372098/17816054/de563abc-663f-11e6-9ab7-acb80d06f498.png)

I did not add it, it’s what is written in the /usr/share/applications/atom.desktop file.

If I just start it from the terminal, it runs perfectly. Both in safe mode or if no arguments are specified.
